pfcpiface/conn.go

```diff
@@ -65,6 +65,24 @@ type PFCPConn struct {
 	hbCtxCancel context.CancelFunc
 
 	pendingReqs sync.Map
+	pConnID     uint32
+}
+
+func (pConn *PFCPConn) GenerateSessionID() uint64 {
```

Comment:
We recently introduced unit tests for upf-epc, every new function should be covered by a unit test. Please implement one.

Reply:
The Test cases have been introduced in the new files that have been added.